agent/exec/container/controller.go

```diff
@@ -196,11 +196,17 @@ func (r *controller) Wait(pctx context.Context) error {
 
 			case "health_status: unhealthy":
 				// in this case, we stop the container and report unhealthy status
+				ctnr, err := r.adapter.inspect(ctx)
+				if err != nil {
+					return errors.Wrap(err, "unhealthy container")
```

Comment:
Sorry here: this error message should say `unhealthy container inspection failed`. Should have brought this up on last review.